packages/util/requirements.txt

```diff
@@ -7,4 +7,5 @@ pyyaml
 requests
 routes
 six>=1.9.0
+SQLAlchemy>=1.4.20,<2
```

Comment:
```suggestion
```

Reply:
I think something went wrong here ?

Reply:
yes, I must have accidentally added it in the subsequent commit.